package.json

```diff
@@ -0,0 +1,84 @@
+{
+  "name": "siddhi",
+  "displayName": "Siddhi",
+  "description": "Intellisense for Siddhi",
+  "version": "1.0.0",
+  "publisher": "siddhi",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/siddhi-io/siddhi-plugin-vscode"
+  },
+  "icon": "resources/images/siddhi.jpeg",
```

Comment:
Check if we can add a png, seems jpeg is curved in the corners and white parts are highly visible